Pose3D[23] limbs #/**< Coordinate frame of the limbs of the robot relative to the robot's origin. */
Vector3 centerOfMass # /**< Position of the center of mass (center of gravity) relative to the robot's origin. 
float32 totalMass  # /**< The mass of the robot. */

